src/java/picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.java

```diff
@@ -25,26 +25,27 @@
         programGroup = Metrics.class
 )
 public class ConvertSequencingArtifactToOxoG extends CommandLineProgram {
-    static final String USAGE_SUMMARY = "Extract OxoG metrics from generalized artifacts metrics";
-    static final String USAGE_DETAILS = "This tool extracts 8-oxoguanine (OxoG) artifact metrics from the output of " +
+    static final String USAGE_SUMMARY = "Extract OxoG metrics from generalized artifacts metrics.  ";
+    static final String USAGE_DETAILS = "<p>This tool extracts 8-oxoguanine (OxoG) artifact metrics from the output of " +
             "CollectSequencingArtifactsMetrics(a tool that provides detailed information on a variety of artifacts found in sequencing " +
             "libraries) and converts them to the CollectOxoGMetrics tool's output format. This conveniently eliminates the need to run " +
             "CollectOxoGMetrics if we already ran CollectSequencingArtifactsMetrics in our pipeline. See the documentation for " +
-            "CollectSequencingArtifactsMetrics and CollectOxoGMetrics for additional information on these tools.<br /><br />." +
-            "Note that only the base of the CollectSequencingArtifactsMetrics output file name is required for the file name input. " +
+            "CollectSequencingArtifactsMetrics and CollectOxoGMetrics for additional information on these tools.</p>." +
+            "" +
+            "<p>Note that only the base of the CollectSequencingArtifactsMetrics output file name is required for the file name input. " +
```

Comment:
I think it would be helpful if you used the actual name of the parameter (INPUT_BASE) instead of writing "input"